File: xcamera/src/main/java/me/shouheng/icamera/listener/CameraVideoListener.java
package me.shouheng.icamera.listener;

import java.io.File;

/**
 * @author WngShhng (shouheng2015@gmail.com)
 * @version 2019/4/14 19:49
 */
public interface CameraVideoListener {

    void onVideoRecordStart();

    void onVideoRecordStop(File file);

    void onVideoRecordError(Throwable throwable);
}

File: icamera/src/main/java/me/shouheng/icamera/listener/CameraVideoListener.kt
package me.shouheng.icamera.listener

import java.io.File

/**
 * @author WngShhng (shouheng2015@gmail.com)
 * @version 2019/4/14 19:49
 */
interface CameraVideoListener {
    fun onVideoRecordStart()
    fun onVideoRecordStop(file: File)
    fun onVideoRecordError(throwable: Throwable)
}